reddit_gold/templates/snoovatar.html

```diff
@@ -64,6 +64,14 @@
       <div class="username">${thing.username}</div>
     %endif
     <p>${_md('view and discuss snoovatars in /r/snoovatars')}</p>
+    %if c.user.gold and c.user.name != thing.username:
+      ${_md('create your own snoovatar [here](/user/%(username)s/snoo)' % dict(username=c.user.name))}
+    %elif not c.user.gold:
+      <div class='snoovatar-footer'>
+        ${_md('create your own snoovatar with [reddit gold](//www.reddit.com/gold?source=snoovatar)')}
```

Comment:
Probably should have mentioned this in chat but my head was not looking at it specifically: You don't actually need the host here - it can just be `/gold?source=snoovatar`. No domain even needed!